crates/moonutil/src/common.rs

```diff
@@ -968,14 +968,15 @@ impl StringExt for str {
 
 pub fn set_native_backend_link_flags(
     run_mode: RunMode,
-    target_backend: Option<TargetBackend>,
+    target_backend: TargetBackend,
     module: &mut crate::module::ModuleDB,
 ) -> anyhow::Result<Vec<String>> {
     let mut all_stubs = Vec::new();
     match run_mode {
         // need link-core for build, test, bench, and run
         RunMode::Build | RunMode::Test | RunMode::Bench | RunMode::Run => {
-            if target_backend == Some(TargetBackend::Native) {
+            if target_backend == TargetBackend::Native {
+                // fixme: does llvm backend also need this? (i assume yes?)
```

Comment:
yes, I have add llvm in https://github.com/moonbitlang/moon/pull/892, need rebase

Reply:
Updated